jekyll/_cci2/testing-ios.md

```diff
@@ -520,17 +501,13 @@ iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current st
 cases CircleCI may not correctly detect the Xcode workspace, project, or
 scheme. Instead, you can specify these through environment variables.
 
-### Constraints on macOS-Based Builds
-There are a few features normally available on CircleCI's standard
-Linux containers that are not available for macOS builds at the moment:
+### Constraints on macOS-based Builds
+
+Splitting tests between parallel containers on macOS is currently not supported.
+We suggest the following strategies:
 
-* Parallelism is not supported
-* While the general `config.yml` file structure will be honored in macOS-based builds
-[configuration options]( {{ site.baseurl }}/2.0/configuration-reference/), the following sections of
-`circle.yml` will not work correctly:
-  * `machine: services`
-  * `machine: <language>`, where `<language>` is any language mentioned
-    in the [Configuration doc]( {{ site.baseurl }}/2.0/configuration-reference/)
+* Using concurrent jobs for building with different Xcode versions
```

Comment:
The existing 2.0 documentation doesn't talk about concurrent jobs (maybe it should?). Is the suggestion here to configure a workflow with parallel jobs? If this is what you mean, it would be better to state it as parallel jobs and point to https://circleci.com/docs/2.0/workflows/#workflows-configuration-examples